src/app/lib/navbar/navbar.component.html

```diff
@@ -1,7 +1,7 @@
 <nav class="navbar" [ngClass]="{'fixed-top': fixedTop}">
     <div #defaultbrand class="navbar-brand" *ngIf="appswitcherbrand.children.length == 0">
         <a [routerLink]="homeUri" class="brand">
-            <img src="./assets/TriFlame.svg">
+            <img height="37px" width="auto" src="./assets/TriFlame.svg">
```

Comment:
that sass change will make it so you dont have to duplicate this inline style...

Reply:
Ah, got it.  Perfect, will do.